app/assets/config/manifest.js

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
maybe i'm just out of scope here, but why this?

Reply:
the Rails default template is this
```
//= link_tree ../images
//= link_directory ../stylesheets .css
```

Reply:
Yes, but we are not using assets, I would like to remove everything related to assets, but if I remove `require "sprockets/railtie"` then I receive this error https://github.com/fgrehm/letter_opener_web/issues/46
Also  Sprockets 4. expect that file https://github.com/rails/sprockets/blob/master/UPGRADING.md#manifestjs

Reply:
aren't we using assets in active admin?

Reply:
I totally forgot that we were using active admin. Good catch